src/shogun/io/SGIO.cpp

```diff
@@ -189,6 +196,7 @@ void SGIO::progress(
 			if (current_val >= max_val-1)
 			{
 				v = 100;
+				last_progress=v-1e-6;
```

Comment:
Yeah, I am still curious about them ;). Regarding IRC, I could be online tomorrow morning, will I find you @karlnapf?